src/main/java/org/eclipse/golo/cli/command/InitCommand.java

```diff
@@ -16,145 +16,138 @@
 import org.eclipse.golo.cli.command.spi.CliCommand;
 
 import java.io.*;
-import java.nio.charset.StandardCharsets;
-import java.util.LinkedList;
-import java.util.List;
+import java.nio.file.*;
+import java.util.*;
 
 import static gololang.Messages.message;
 import static gololang.Messages.info;
 
+// TODO: add project files:
```

Comment:
It was unclear for me if it's a wanted behavior. I'll remove the note and open an issue to allow discussion.